homeassistant/components/zha/core/channels/general.py

```diff
@@ -332,11 +335,40 @@ class Partition(ZigbeeChannel):
     pass
 
 
+@registries.CHANNEL_ONLY_CLUSTERS.register(general.PollControl.cluster_id)
 @registries.ZIGBEE_CHANNEL_REGISTRY.register(general.PollControl.cluster_id)
 class PollControl(ZigbeeChannel):
     """Poll Control channel."""
 
-    pass
+    CHECKIN_INTERVAL = 55 * 60 * 4  # 55min
+    LONG_POLL = 6 * 4  # 6s
+
+    async def async_configure(self) -> None:
+        """Configure channel: set check-in interval."""
+        try:
+            res = await self.cluster.write_attributes(
+                {"checkin_interval": self.CHECKIN_INTERVAL}
+            )
+            self.debug("%ss check-in interval set: %s", self.CHECKIN_INTERVAL / 4, res)
+        except (asyncio.TimeoutError, zigpy.exceptions.ZigbeeException) as ex:
+            self.debug("Couldn't set check-in interval: %s", ex)
+        await super().async_configure()
+
+    @callback
+    def cluster_command(
+        self, tsn: int, command_id: int, args: Optional[List[Any]]
+    ) -> None:
+        """Handle commands received to this cluster."""
+        cmd_name = self.cluster.client_commands.get(command_id, [command_id])[0]
+        self.debug("Received %s tsn command '%s': %s", tsn, cmd_name, args)
+        self.zha_send_event(cmd_name, args)
+        if cmd_name == "checkin":
+            self.cluster.create_catching_task(self.check_in_response(tsn))
+
+    async def check_in_response(self, tsn: int) -> None:
+        """Respond to checkin command."""
+        await self.checkin_response(True, 8, tsn=tsn)
```

Comment:
What does 8 mean here? Can we add a constant so we don't just have a magic #?

Reply:
Fixed